modules/api-workbench-creating.adoc

```diff
@@ -0,0 +1,379 @@
+:_module-type: PROCEDURE
+
+[id='api-workbench-creating_{context}']
+= Creating a workbench by using the `Notebook` CRD
+
+[role="_abstract"]
+
+In {productname-short}, you can create a workbench object by using the `Notebook` Custom Resource Definition (CRD). 
+
+In the following procedure, you configure a `Notebook` CRD and then use it to create the `Notebook` Custom Resource (CR) that defines the workbench.
+
+.Prerequisites
+
+* You have cluster administrator privileges for your {openshift-platform} cluster.
+* You have installed the OpenShift command-line interface (CLI). See link:https://docs.redhat.com/en/documentation/openshift_container_platform/{ocp-latest-version}/html/cli_tools/openshift-cli-oc#installing-openshift-cli[Installing the OpenShift CLI^].
+* You have created a data science project. In the example in this procedure, the project is named `my-data-science-project`.
+
+//will probably need to fix these links
+ifdef::upstream[]
+* You know the URL for the notebook image that you want to use in the workbench. The example is this procedure uses the custom image that you created in link:{odhdocshome}/api-workbench/#api-custom-image-creating_api-workbench[Creating a custom image by using the `ImageStream` CRD]. 
+endif::[]
+ifndef::upstream[]
+* You know the URL for the notebook image that you want to use in the workbench. The example is this procedure uses the custom image that you created in link:{rhoaidocshome}{default-format-url}/api_workbench/api-custom-image-creating_api-workbench[Creating a custom image by using the `ImageStream` CRD]. 
+endif::[]
+
+.Procedure
+
+. In a terminal window, if you are not already logged in to your {openshift-platform} cluster as a cluster administrator, log in to the OpenShift CLI as shown in the following example:
++
+[source,subs="+quotes"]
+----
+oc login __<openshift_cluster_url>__ -u __<admin_username>__ -p __<password>__
+----
+
+. Define the `Notebook` CRD.
+
+.. Create a YAML manifest file named `notebook.yaml`.
+.. Copy the following configuration and paste it in the `notebook.yaml` file:
++
+.Example `Notebook`
+[source,yaml]
+----
+apiVersion: kubeflow.org/v1
+kind: Notebook
+metadata:
+  annotations:
+    notebooks.opendatahub.io/inject-oauth: 'true' # <1>
+    opendatahub.io/image-display-name: Minimal Python # <2>
```

Comment:
This is where I got confused - I thought the example would be "My custom image" - why is it now Minimal python?